internal/convert/cluster_config_test.go

```diff
@@ -170,9 +175,13 @@ func TestClusterConfig_PatchAutomationConfig(t *testing.T) {
 								DBPath: "/data/db/",
 							},
 							SystemLog: opsmngr.SystemLog{
-								Destination: "file",
+								Destination: file,
 								Path:        "/data/db/mongodb.log",
 							},
+							AuditLog: &opsmngr.AuditLog{
+								Destination: file,
+								Path:        "/data/db/audit.log",
+							},
```

Comment:
[q] is there any reason why you are using `file` instead of `"file"`?

Reply:
`file` is a constant with `"file"` in it, I guess I should make all of these use the constant